app/controllers/org_chart_controller.rb

```diff
@@ -0,0 +1,69 @@
+class OrgChartController < ApplicationController
+  before_action -> { user_has_position("Secretary") }, only: [:create, :destroy]
+  before_action :authenticate_user!
+
+  def index
+    @charts = []
+    division_names = User.uniq.pluck(:division).reject(&:nil?)
+    division_names.each do |name|
+      data_table = GoogleVisualr::DataTable.new
+      data_table.new_column('string', 'Name'   )
+      data_table.new_column('string', 'Manager')
+      data_table.new_column('string', 'ToolTip')
+      data_table.add_rows(get_array_for_division(name))
+      options = { allowHtml: true }
+
+      chart = GoogleVisualr::Interactive::OrgChart.new(data_table, options)
+      @charts.push(chart)
+    end
+  end
+
+  def admin
+    @divisions = User.uniq.pluck(:division).reject(&:nil?)
+  end
+
+  def create
+    user = User.find_by(id: params[:user][:existing_id])
+    params[:user][:division] = params[:user][:division].present? ?
+                                               params[:user][:division] :
+                                               params[:user][:existing_division]
+    if user
+      user.update(user_params)
+    else
+      user = User.new(user_params_with_name)
+      user.status = 'org_chart_dummy'
+      user.email = ('a'..'z').to_a.shuffle[0,15].join
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.